openeo/__init__.py

```diff
@@ -17,11 +17,28 @@ class BaseOpenEoException(Exception):
 from openeo.rest.connection import connect, session, Connection
 from openeo.rest.job import BatchJob, RESTJob
 from openeo.internal.graph_building import UDF
-
+import requests
+import warnings
 
 def client_version() -> str:
     try:
         import importlib.metadata
         return importlib.metadata.version("openeo")
     except Exception:
         return __version__
+
+def check_if_latest_version():
+    try:
+        package = 'openeo'
+        response = requests.get(f'https://pypi.org/pypi/{package}/json')
+        latest_version = response.json()['info']['version']
+    except:
+        # Probably no internet connection available, pass
+        return
+    installed_version = client_version()
+    if latest_version != client_version():
```

Comment:
Why a `!=`  check instead of `>` ?

for developers the local version could be ahead of the latest published version and you don't want this warning then

Reply:
I'm using the ComparableVersion class from openeo.capabilities now.